run/lib/case.libsonnet

```diff
@@ -61,6 +61,11 @@
       '/bin/vbank',
       '-cluster-name=%s' % args.clusterName,
     ],
+  crossregion(args={ tso_request_count: '100' })::
```

Comment:
Does this need to be modified to 2000 too?

Reply:
updated